### PR TITLE
Add <script href> as a viable attribute for dependency detection

### PR DESCRIPTION
**Pull request by kirillrogovoy**
# ↪️ Pull Request

Closes https://github.com/parcel-bundler/parcel/issues/3572.

## 💻 Examples

Please, see the original issue.

## ✔️ PR Todo

- [x] Added/updated unit tests for this change
- [x] Filled out test instructions (In case there aren't any unit tests)
- [x] Included links to related issues/PRs

### CODE REVIEW

File: packages/transformers/html/src/dependencies.js [hunk 1]
```diff
@@ -18,7 +18,8 @@ const ATTRS = {
     'iframe',
     'embed'
   ],
-  href: ['link', 'a', 'use'],
+  // Using href with <script> is described here: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/script
+  href: ['link', 'a', 'use', 'script'],
   srcset: ['img', 'source'],
   poster: ['video'],
   'xlink:href': ['use', 'image'],
```
Comment (devongovett) [inline packages/transformers/html/src/dependencies.js]: Do we need to add `'script'` here as well? Looks like SVGs support both `href` and `xlink:href`.

Comment (kirillrogovoy) [reply packages/transformers/html/src/dependencies.js]: Also linking the same suggestion by @andersk — https://github.com/parcel-bundler/parcel/issues/3572#issuecomment-538673727

I think you guys are right. I didn't bother to check it and assumed that SVG2 is the current recommendation.

OMW fixing it!

Comment (kirillrogovoy) [reply packages/transformers/html/src/dependencies.js]: Done. :tada:

**Comment by devongovett** [comment]
Thank you!

**Pull request merged**